CMSgov/mint-app
# [EASI-2666] Updated existing_models Data to Current Spec

# EASI-2666

## Changes and Description

- Added migration to update existing_models table with current data



## How to test this change

Run scripts/dev db:seed and observe that the existing_models table is populated with current data

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
